package.json

```diff
@@ -99,7 +100,8 @@
     "typescript": "^3.8.3"
   },
   "dependencies": {
-    "leaflet": "^1.6.0"
+    "leaflet": "^1.6.0",
+    "@googlemaps/js-api-loader": "^1.14.3"
```

Comment:
This should be a peer dependency, it's only needed for those that use the google provider.

Reply:
Great! Fixed in d58c3634d632fe8fa002c2a28dcb!